src/ConnectedMode.UnitTests/Suppressions/ServerSuppressionsChangedHandlerTests.cs

```diff
@@ -20,7 +20,6 @@
 
 using System;
 using SonarLint.VisualStudio.ConnectedMode.Suppressions;
-using SonarLint.VisualStudio.Core.Suppression;
```

Comment:
Miscellaneous: redundant using.

FYI the class and tests are essentially a copy of this test and associated class but with the different event sources and targets.

The `ServerSuppressionsChangedHandler` listens for updates to the list of server suppressions the client knows about and tells the `ClientSuppressionSynchronizer` to synch the local issues and server issue suppression states.

The new `LocalSuppressionsChangedHandler` listens for notifications from the  `ClientSuppressionSynchronizer` that something changed as a result of the synch, and triggers and update of the UI by calling `IIssueLocationStore.Refresh(files)`.